### PR TITLE
Removes cat things from codebase

### PR DESCRIPTION
**Pull request by MadmanMartian**
My apologies for the inconsistent whitespace [I got you -- Editor], I don't believe my preferences are set up appropriately. If I have missed anything, kindly point it out.

This shouldn't fuck up peoples preferences, as there is some sanity handling if you've selected a species that doesn't exist

```dm
    var/datum/species/chosen_species
    if(!roundstart_checks || (pref_species.id in GLOB.roundstart_races))
        chosen_species = pref_species.type
    else
        chosen_species = /datum/species/human
        pref_species = new /datum/species/human
        save_character()
```

roundstart_races is populated through

```dm
/proc/generate_selectable_species()
    for(var/I in subtypesof(/datum/species))
        var/datum/species/S = new I
        if(S.check_roundstart_eligible())
            GLOB.roundstart_races += S.id
            qdel(S)
    if(!GLOB.roundstart_races.len)
        GLOB.roundstart_races += "human"
```

:cl: 
del: Removed Felinids
/:cl:

[why]: # (Please add a short description [two lines down] of why you think these changes would benefit the game. If you can't justify it in words, it might not be worth adding.)
There was a vote

### CODE REVIEW

**Comment by MadmanMartian** [comment]
But the existing one also completely removes laser pointers

File: code/game/objects/items/devices/laserpointer.dm [hunk 1]
```diff
@@ -131,7 +131,7 @@
 
 	//catpeople
 	for(var/mob/living/carbon/human/H in view(1,targloc))
-		if(!iscatperson(H) || H.incapacitated() || H.eye_blind )
```
Comment (Qustinnus) [inline code/game/objects/items/devices/laserpointer.dm]: this entire block of code needs to be removed, by doing this you're making it so all people are affected by laser pointers

Comment (MMMiracles) [reply code/game/objects/items/devices/laserpointer.dm]: we're all part cat on the inside

Comment (MadmanMartian) [reply code/game/objects/items/devices/laserpointer.dm]: Well that was a hilarious oversight

**Comment by Iamgoofball** [comment]
It doesn't completely remove laser pointers though?
Check the diff again.
https://github.com/tgstation/tgstation/pull/40707/files#diff-a1480776ef0f5634a7b98dfc077ca6caR1

**Comment by Qustinnus** [comment]
@Iamgoofball this one is still open for comments though which is way funnier because shezza might get mad again

**Comment by obscolene** [comment]
Hell yes. Hell fucking yes.

**Comment by MadmanMartian** [comment]
@Iamgoofball well now. When I saw that there was so many lines removed, I guess I sorta blanked over it.

**Comment by Iamgoofball** [comment]
Sorry about the confusion. If you want to keep it open I guess you can but it'll prolly just get closed for being a dupe of the original.

**Comment by Qustinnus** [comment]
image
top 10 anime plot twists

**Comment by Anonymous583** [comment]
been blueballed on this merge for too long, do itttt

**Comment by bobbahbrown** [comment]
i cannot express my excitement for these new changes -- finally some fresh air!!

**Comment by Qustinnus** [comment]
image
dab on the cat

**Comment by Therapist1** [comment]
after they get removed, what do you think the new race will be? my money is on space orcs.

**Comment by Iamgoofball** [comment]
space orks would be sick as fuck but i can hear the games workshop lawyer sirens on the horizon already

**Comment by Anonymous583** [comment]
just make them blue instead of green

**Comment by Qustinnus** [comment]
https://streamable.com/ffxhh I still have my WIP race which kinda resembled zoltans in terms of look but instead theyre basically moving torches that display their health by being a certain color (green = healthy, red is fucked)

also they do burn damage when punching shit

**Comment by MadmanMartian** [comment]
Looks cool

**Comment by Whoneedspacee** [comment]
Go ahead and merge this one if you wanna as long as it works, whatevers.

**Comment by Iamgoofball** [comment]
@Whoneedspacee This PR adds completely un-needed code to fix a bug that didn't exist.

**Comment by Iamgoofball** [comment]
wait no this doesn't even add code this is a direct duplicate of the other pr with a different change to laser pointers, wew

**Comment by MadmanMartian** [comment]
@Iamgoofball ??

**Comment by Iamgoofball** [comment]
This PR doesn't do anything new compared to the other catperson removal. I don't see why it needs to exist when the other one does the job fine.

**Comment by Iamgoofball** [comment]
fair and balanced

**Comment by FantasticFwoosh** [comment]
> 
> 
> But the existing one also completely removes laser pointers

That's a fair point, maintainers should force changes on that PR or use this one else. Laser pointers pre-date cats and the code to make them jump chasing it should be removed instead of the item, laser pointers are still valuable tools for pseudo pointing, blinding & knocking out borgs especially at high laser levels after being upgraded.

**Comment by FantasticFwoosh** [comment]
> 
> 
> after they get removed, what do you think the new race will be? my money is on space orcs.

Blood Elf space clowns, i called my money on it long ago. Differentiate clowns as a different race.

**Comment by Iamgoofball** [comment]
@FantasticFwoosh did you even read the comment thread

**Comment by zamolxius** [comment]
praise the lord

**Comment by GuyonBroadway** [comment]
Add back the laser pointer thing as a negative quirk.

**Comment by evsey9** [comment]
replace catgirls with vulpkanines

**Comment by BeeSting12** [comment]
The vote was actually for removing them as a chooseable species. They should remain in the codebase, just not as a chooseable species.

**Comment by GuyonBroadway** [comment]
Funny how nobody bought that point up before now.

**Comment by Bawhoppen** [comment]
It's understandable nobody noticed. The people who were ss13 political junkies already knew what the vote was about, so they didn't read the actual wording very carefully. 
After it was over, everyone came to review the poll, and only then did they notice the specific wording.

I see it as a very reasonable assumption that "remove as a choose-able species" and "remove entirely from the code" would be viewed by the silent majority voters differently, and would receive different results.

**Comment by Bawhoppen** [comment]
@GuyonBroadway That's true, but I think it's probably going outside of the bounds of the wording.
Take for example the recent referendum in Romania that tried ban gay marriage:
Assume the wording was as simple as (which it was not IRL) "do you want gay marriage to be illegal?" Imagine if it passed, and then the political leaders responded by banning *ALL* marriage. That would make gay marriage illegal too, but is obviously a clear overreach of the intended wording that the voters went by.

I believe that it is in essence the same *type* of issue as here. 
The wording "Do you want felinids to remain in /tg/ as a chooseable species?" uses the word "chooseable"; something which I think implies that we're just deciding whether to have it remain as being chooseable or not. 
I don't think it can be denied that if the question was instead, "Do you want felinids to remain in /tg/ as a species?", it would have a different meaning.

Ultimately, I think it is probably something worth being considered and brought up.

**Comment by Iamgoofball** [comment]
it's getting removed lmao, the poll was to remove from codebase, and it won

get over it lmao

**Comment by Militaires** [comment]
image
MEW MEW MEW MEOW! +1 I would rather have mutant races though, because you can change the saymod to "meows" and "purrs", and slash instead of punch. Either way, I love this, and every anti-lizard biggot is going to catch fire when they see this!

**Comment by BeeSting12** [comment]
@Iamgoofball 
I'm pretty sure we used the same line on you guys when we had the poll a year and two years ago, but you didn't get over it. If screaming autistically worked for that then it'll probably work here, it'll just take a couple years.

**Comment by ninjanomnom** [comment]
`git cherry-pick useful-support`

**Comment by Qustinnus** [comment]
@BeeSting12 except autistically screeching /didnt/ work because catgirls stayed.

**Comment by BeeSting12** [comment]
It did work. They're getting removed, it just took a while.

**Comment by Qustinnus** [comment]
i'm sure that was just the autistic yelling that got the poll in our favor

**Comment by MMMiracles** [comment]
the separation goes both ways as it turns out

**Comment by GuyonBroadway** [comment]
@BeeSting the issue here is opinions change, people were once tolerant of catgirls but have now grown weary of the antics of the people they attract. You can blame the likes of shezza and wilchen when they "ironically" spam post shitty memes in the face of "hates" because "HAH WE WON A POLL LOLOLOLOL GET TRIGGERED, DABMINT ON YOU!" and people who were neutral got fed up with it and wanted them gone because this helpies crap isn't funny anymore.

Its funny because its not even in game shitlery that was the issue, it was mostly ooc crap that stood out. Combined with shit like handholding and headpatting prs it became hard to seperate those few pixels from what was the thorn in the side of the tg community. 

You can argue that you'd upset half the server if you remove them, but by the same token you'd upset half the server if you didn't. Why does one half gain precedence over the other?

Now you can argue that if opinions change then they can logically change back. So it seems fair that somewhere down the line you hold another vote, 6 or so months (considering the last poll was over a year ago this is more than fair) and poll again and see what the community thinks after its had time without catgirls, after the memory of shitposting softcore catgirl porn rather that debating an issue has faded.


As a seperate question not strictly related to my previous points I must ask. Lets pretend catgirls are not in the code and never have been. Why do you think they should they be added? What do they bring to ss13?

**Comment by Qustinnus** [comment]
did you just say..shezza and wilchen?


theyre the same person my guy

**Comment by epochayur** [comment]
@BeeSting12 there are plenty of other servers to harvest your furry cummies from, the people have spoken

**Comment by Whoneedspacee** [comment]
@CosmicScientist It’s like you didn’t even read the original PR and my arguments. My main point was they’re basically a human with ears and tail and there’s no niche they fit in ss13 other than fetish.

At least other content like lizards that other people describe as being fetishy have some place in ss13, catgirls are just shitty spam sexual memes and people are tired.

Tell me the unique niche that felinids filled in the codebase and what they added to rp besides shitty memes. They were a terrible addition to the game. It’s almost wonderful though that people are actually making new and good races on the brink of felinids removal so maybe we’ll actually see some better ones now.

**Comment by GuyonBroadway** [comment]
@cosmicscientist

>>being triggered by seeing ies added onto help

Its exactly this kind of memey dismissivness that people are fed up with. Rather that actually talk to people, dabbermint gets broken out and questions ignored.

>harassing contributors on their PRs

Aside from joking calling config options kevinz fetish that one time when did I ever do that?

>Usually if only half of everyone isn't backing the status quo

By the same token only half of people backed adding them in the initial pr. There was limited support for them and they were added with the caveat that they would be adminspawn only because it was a pointless feature that was the subject of contention even then.

https://cdn.discordapp.com/attachments/276455180852264960/502353494255075329/Screenshot_2018-10-18-06-30-20.png

The new poll is because opinions do change, don't be so short sighted to think otherwise, once the people who respond to criticism with animelol.gif have gone maybe a more civil discussion on the subject can be had and perhaps someone can justify adding a few pixels to somones head without their argument boiling down to "ur just upset."

That does seem to be the common stance here sometimes. "The more people critique and dislike something the more WRONG they must be!" considering the amount of times stuff has been merged despite overwhelming negative feedback.

**Comment by TheRespectedMan** [comment]
The citadel station invasion has been  r e p e l l e d 
Back to the BYOND hub felinids, back to citadel and paradise station, begone!

There is a server **_specifically designed_** for those who want to play whatever fetish race they want, it's called "paradise station". Make a copy paste felinid PR and paradise will speed merge it guaranteed. 
Felinids will make an excellent fit with the sentient fursuit race they call "vulpkanin" on paradise station. 
They will be welcomed as family.

**Comment by CosmicScientist** [comment]
@GuyonBroadway tell me what's wrong with helpies because unless I'm mistaken, one admin checked the logs for a round and found, without it being said, one player was angry at helpies. If you're honestly triggered by someone adding RP to their words, make a policy thread or better yet, play on your own downstream where there are no cats.

m'lord don't you remember when you went arse over tits over sprites?

[Digitigrade shoes #40514](https://github.com/tgstation/tgstation/pull/40514)

>Closing it isn't enough, gotta smear its drained walls in poop to make sure nobody tries to break in at night and go skinny dipping.

>As a coder, snowflake code drives me into an inconsolable rage, especially snowflake code for a trivial pointless snowflake feature.

Charge ur phone

>By the same token only half of people backed adding them in the initial pr. There was limited support for them and they were added with the caveat that they would be adminspawn only because it was a pointless feature that was the subject of contention even then.
The new poll is because opinions do change, don't be so short sighted to think otherwise, once the people who respond to criticism with animelol.gif have gone maybe a more civil discussion on the subject can be had and perhaps someone can justify adding a few pixels to somones head without their argument boiling down to "ur just upset."

I don't know what this is trying to say. "There was half support then. It was only added because it was adminspawn. Opinions change. We should remove it because we should treat it like a new feature we don't already have." Did I get it right? Please clarify because I don't understand what it's trying to argue.

I don't know why you're donning a persecution complex on your last paragraph. Your total critique has been "I get triggered by peppermint, I get triggered by code PRs on a codebase I'm not a part of, I get triggered by someone typing out helpies. I get triggered by plushie relationships. Let's say catgirls need to be justified in being added again because I can't make an argument for removing them."

**Comment by GuyonBroadway** [comment]
@CosmicScientist 

"Helpies" is as much rp as surounding everything you type with tildies to make yourself come across like an anime character. And when it's not that it's the one word a previously silent catgirl bartender says to instantly summon their equally silent catgirl buddies to help them out in a fight. And before you joke about that copypasta that story I posted was a recounting of a genuine attempt to see if the reddit memes were true or not, i got what was practically a metasquad for my efforts.

Snowflake code is bad, if in order for your shoes of all things to work you need to add extra lines of code to check for specific circumstances where they can/cannot outside of existing be worn your idea needs rethinking.

To clarify, this poll was held and it shows opinions on server have shifted from "overwhelmingly in favour of catgirls" to "basically 50/50" and I was acknowledging that opinions could indeed change again later so furthur polls later would be in order.

Me asking BeeSting was more a curiosity than anything in that a lot of arguments for cats is "its just a few pixels" and taking the stance that they are already here why bother? One thing I never heard was why they were added in the first place besides maybe"owned the liggerhaters!" like did I miss a page or something? Fill me in here!

The getting "triggered" by dabbermint is more that if you ask a catperson to be less obnoxious what do you get? You can't talk to them or ask them to maybe dail the cringe back a bit. We coulda been reasonable about it but hey ho you posted dabbermint again and said "lol triggered" sobI guess discussion on the subject is out the window. The growing anti-cat sentiment from this basically boils down to being fed up with being brushed off like this and the only voice left was via github. 

Funny how the dabbermints went away and the "we need a compromise solution" only came out when the threat of removal was actually on the table. If catgirl players weren't insufferable wazzocks, i wouldn't be advocating thier removal.

And yes, while it seems unfair to press an issue onto a large chunk of the players for the actions of a few but I've not seen any level of cliqey cringe nonsense like I have with catgirls and I have to ask if its worth enabling this kind of behaviour for the sake of a few pixels.

**Comment by CosmicScientist** [comment]
If you dislike "Helpies" and tildes, why aren't you playing on your downstream server or *ahelping over failRP?*

If you can see metacomms, why aren't you adminhelping?

I don't know what copypasta you're talking about. Are you using one in this srs PR?

Snowflake code is everywhere. With your excessive use of *snowflake liggers* and complaints about minority players, I assume you had more against the PR than it using a special check.

Thank you for clarifying. I don't know what to say about that though.

I don't know why they were added in the first place, the addition PR has 200+ comments I don't want to read through but looking close to the merge action, even oranges appeared fine with it at the time.

Telling me a catgirl player is stonewalling you with gifs isn't much to cry over. If it's on GitHub, call in a maintainer to clean up the spam or a moderator if it's the forums and not in the hut. If it's a headmin or maintainer *and you're talking about a serious issue,* drag their name through the muck. If it's just you and them in forums PMs or a discord channel, I don't know what you expect since neither are formal places with standards besides what the server host or discord group permits. I also don't know why you think you're being brushed off - if your problems are players and not host/headmins/maintainers then you're not being brushed off because by what authority or power does the gif spammer have to act on your wishes?

I also don't know why you think it's funny that it went from dabbermint to compromise when, as far as I'm aware, Shezza is not me. I haven't heard Shezza asking for a compromise and I assume he'll still dab on you. And I assume the headmins making a config change had not dabbed on you either. So you can stop playing the victim card when you're the one going to Shezza to be dabbed on, when he holds no power or authority besides owning a folder of Peppermint memes.

You probably weren't around for lizard metagangs if you think the catgirl clique is concerning. And that is again an admin issue. Players can be banned on a rule zero if they're being excessively terrible.

To summarise the entirety of the above in case you want to refute me without us making more text walls: Why aren't you ahelping, reporting and blocking piss poor behaviour? Why are you supporting a code change that neither bans bad players nor affects administration but gets rid of a thing because people you don't like happen to like it?

**Comment by Iamgoofball** [comment]
>If you dislike "Helpies" and tildes, why aren't you playing on your downstream server or ahelping over failRP?

Possibly because the admins aren't going to do anything? They actively participate in this crap, and FailRP only applies when they don't like your name.

**Comment by duncathan** [comment]
I firmly disagree with using a tg server poll to make codebase decisions when tg isn't the only server that uses pure tg code

if players vote in favor of removing something which is literally a config option then it should be removed from the config not the codebase imoimo

**Comment by evsey9** [comment]
tg players play on the code
other servers have full control of the code
If they want to they can simply revert this change or not merge it
This isn't like some big change that makes it very hard on the code maintainers if they decide not to merge

**Comment by duncathan** [comment]
I'm talking about non-tg servers that use unmodified tg code. The popular downstreams are not my concern; they're equipped to deal with such things

**Comment by evsey9** [comment]
So what you're saying is we gotta run every change through every server host that might run unmodified tgcode?

"B-But other servers might not like thiiiiiis!" is not a good excuse when this is a codebase that is most used on the tgstation server and even HAS THE SAME NAME. GEE, I WONDER WHY?

**Comment by evsey9** [comment]
Why is this still open if the correct filtered vote results show that more than 50% are in favor of the species remaining

**Comment by Bawhoppen** [comment]
@evsey9 oranges has said they are going to conduct one last final poll ever (TM), to finally decide it.

**Comment by GuyonBroadway** [comment]
I'm sure oranges has a reason. Much the same reason why the tg headmins were resistant to removing them before the filter was supposedly fixed.

**Comment by MMMiracles** [comment]
as long as the second poll actually states it'll be a full removal instead of just disabling them I don't see an issue.

**Comment by PKPenguin321** [comment]
>If you dislike "Helpies" and tildes, why aren't you playing on your downstream server or ahelping over failRP?

Your argument isn't unique to your side. If you like cats so much, why not play a downstream where they're encouraged?

**Comment by Iamgoofball** [comment]
@evsey9 Even though that more than 50% is now up to just 51%?

**Comment by ExcessiveUseOfCobblestone** [comment]
the vote is final g-g-g-guys

**Comment by CosmicScientist** [comment]
@Iamgoofball the codebase is for code, the administration is for policy. If the admins see "Helpies" and tildes as fine then this PR doesn't change those people unless you plan on hardcoding ckey blockers or censoring words.

@PKPenguin321 I imagine cat players can hop onto citadel, vore or a downstream that won't merge this change. My argument is unique given that my target is Guyon whose argument is:

- my server didn't enable the config defaulted to off, so I want a feature I don't see removed
- I won't click block when Shezza shitposts, so I want their feature removed
- I think there's a conspiracy where catgirls caused plushie relationships and other PRs I disagree with

If you bring up any more valid arguments like this is neither the first nor last bastion of catgirls, I'll agree with them because catgirls aren't a feature I use and I'd love a sane person to talk to.

**Comment by Militaires** [comment]
i expect catgirls to win the poll this time tbh, one-vote-difference miracles like that dont happen twice, do they?

**Comment by GuyonBroadway** [comment]
So i've been playing ss13 for a long time, back when we had word learnin' cult, robotics was down next to engineering, the sing deleted turfs and was laggy as balls and all we had was boxstation. Hell I remeber when the AI had to use a drop down list to navigate the fucking station. 

Never in all my time playing ss13 have I seen anything quite like what came around with the addition of catgirls. Like, handholding, snowflake checks for digitgrade boots, skirts for sec, Handholding, headpatting, blob cuddles and so fourth, all that came after the proverbial floodgates opened. Not all of these can be disabled downstream.

Hell before catgirls, if you had tried to add "Mehki goes on the table" or even put that as a joke line in a PR you'd be laughed at for trying to add something cringeworthy to the game that wasn't a badly written library book. 

IC too, I never saw metabuddies quite like catperson metabuddies, metafreinds tend to be quiet dicks who use discord telepathy to help eachother out. With catgirls you'll suddenly see an engineer, a chemist and botanist with shit they built in science going around in a fucking pack. You even see stuff like the captain holding a fucking TEA party with their "pet" [(who has guns from the armoury no less)  ](https://youtu.be/VuOn96y595A?t=75)

Sure it may seem like "just a few bad people" but these few bad people only turned up/started being bad when catgirls were added and are the ones who cry loudest when catgirls (and nothing else mind you) are under threat. So call me mad, call me a conspiritard, say its "not my server not my problem" but this is what I've seen.  

It was never just about a few pixels, if it were I'd probably be laughing at the sidelines too, its the the kind of behaviour it encourages and attracts and I've never seen ANYTHING else have quite the same effect on a playerbase.

**Comment by evsey9** [comment]
the gates have already been opened
removing catgirls won't accomplish much
tg is diseased -- rotten to the core, there's no saving it. We need to pull it out by the roots. Wipe the codebase clean - burn it down. And from the ashes a new /tg/station will be born. Evolved but unfetishized. 
The weeb will be purged, and strongest will thrive; free to code as they see fit. They'll make /tg/station great again.

**Comment by Qustinnus** [comment]
and its not even that its the coders who are being degenerates. Most of the coders are either neutral or against catgirls and the only reason they're not out yet is because people think that cat players leaving is unironically a problem.

**Comment by KetasDotepasto** [comment]
I seriously don't understand this animosity towards catpeople. 

Furthermore, if catpeople are removed, human players can no longer ask the AI to kill all of them. Committing genocide without causing collateral damage is a very though provoking exercise, which I believe should remain in the game. While this genocide could be done on Lizards, catpeople are the more challenging target, since they have an almost human appearance.

**Comment by CosmicScientist** [comment]
tfw Guyon is pouring his heart out against cattegrilles because he can't code out skirts. Are the rest of his downstream's maintainers as incompetent?

**Comment by YakumoChen** [comment]
The poll is pretty blatantly solvable by configuration options, the question "chooseable as a species" does not require an edit to the code.

**Comment by evsey9** [comment]
if you wouldn't be able to be a catgirl either way, no matter config or code change, then why would you care about it being a code change, hm? Why keep code on the code base if it's not used?

**Comment by MMMiracles** [comment]
leaves the species in for admin memes and so i can make my fnaf space ruin featuring furrzy the animatronic catgirl

**Comment by AffectedArc07** [comment]
***M   E   R   G   E***

**Comment by actioninja** [comment]
No matter what happens, this will be hilarious.

**Comment by MadmanMartian** [comment]
Is this new vote up yet? It doesn't seem to have been linked anywhere for people to vote on it if so

**Comment by ExcessiveUseOfCobblestone** [comment]
Just FYI admins will just pseudo give roundstart cat species via purrbation so making it admin only is basically making it roundstart but feeling naughty when you're doing it.

**Comment by Qustinnus** [comment]
@MadmanMartian votes are ingame

**Comment by Qustinnus** [comment]
higher visibility for people that dont actually interact with the server beyond playing

**Comment by optimumtact** [comment]
we're still deciding on the votes wording

**Comment by Iamgoofball** [comment]
You're going to keep getting dicked around until voter fatigue or they can decide on maximum voting options to make sure they can increase voter fatigue.

**Comment by Iamgoofball** [comment]
Especially considering active admins are in here saying they'll subvert any non full removal poll.

**Comment by MadmanMartian** [comment]
I'm just surprised at the apparent divide between the administration and the coder team on tg.

**Comment by Iamgoofball** [comment]
The admin staff do not work for anyone's benefit but their own.

**Comment by MMMiracles** [comment]
I fucking *HATE* ADMINS SO GOD DAMN MUCH.

**Comment by Supermichael777** [comment]
Technically all player referendums are non binding. A B S O L O U T E

**Comment by Anonymous583** [comment]
The insistance from the admins on the second pull reeks of desperation, and I refuse to believe it takes more than ten minutes to agree on wording let alone a day. This should have been merged and put to bed for good days ago.

**Comment by Allspice** [comment]
@ExcessiveUseOfCobblestone Someone may well have mentioned it in the previous hundred posts, but I wonder: Did anyone object to the wording before they lost the vote? Because it smells a bit like, "hold 67 votes that aren't official enough but the 68th is official and good and just happened to coincidentally rule in my favour."

**Comment by Kawaii-Big-Boss** [comment]
Doesn't matter whether they won or not. What needed to be seen was how high the divide was. A whopping 50%.
After that point what should have been done in the poll is ask people questions similarly to what they did, but for both sides instead of just the removal side. It would be better to determine if people hated them for not adding anything to the game, hated them for being degenerate, or didn't care either way. Same goes for the other side, it's better to see if the "Yes" voters voted "yes" simply because they didn't care, or if it legitimately would result in a ton of people leaving.
That would be the deciding factor after that point, if you ask me. Even though I'm fully aware that had the votes even been a little farther apart, filters applied wrong or not, the PR would have instantly been closed and no discussion would have been had afterwards.

**Comment by SonixApache** [comment]
TG has the most fun PR threads

**Comment by ExcessiveUseOfCobblestone** [comment]
@Allspice I don't think so, but I think the timing of the argument shouldn't be used to shut down the argument.

It WAS poorly written compared to what it wanted to do, and that's coming from someone who wants it removed.

**Comment by optimumtact** [comment]
See - https://tgstation13.org/phpBB/viewtopic.php?f=40&t=20294

**Comment by Iamgoofball** [comment]
still should be merged
